corehq/apps/app_manager/templates/app_manager/partials/modules/module_view_settings.html

```diff
@@ -49,28 +49,29 @@ <h4 class="panel-title panel-title-nolink">{% trans "Menu" %}</h4>
           <div class="panel-body">
             {% if not module.is_surveys and module.module_type != 'shadow' %}
               {% include "app_manager/partials/modules/module_view_case_type.html" %}
-            {% endif %}
```

Comment:
Does this change affect whether or not this feature is available for shadow modules?

Reply:
Yes, I changed it because I observed that the case_type setting was also under that [conditional](https://github.com/dimagi/commcare-hq/pull/29900/files#diff-d59af1ba8ae486c6decb9b89542e459d2adc916a787d971f97d2b13387b6101dR51)

Reply:
Got it. The case type for shadow modules gets inherited from the source module, [here](https://github.com/dimagi/commcare-hq/blob/0728e983d89ac19631a09d0bae22188e90486376/corehq/apps/app_manager/models.py#L3722-L3726), so if the additional case types can't be configured on a shadow module, they'd need to similarly inherit from the source module.

I think it's okay to implement whichever is simpler. My intuition is that shadow modules will use the same additional_case_types as the source modules but that it's not a big deal for app builders to have to specify it, especially since we have the functionality to copy an entire case list/detail config from one module to another.